tests/test_controllers/MainConfig_test.py

```diff
@@ -114,17 +114,17 @@ def test_yaml_config(self):
                 new=mock.mock_open(read_data=yaml_config))
     @mock.patch.dict(os.environ, {'FTS_SSLCOT_PORT': '10000'}) # int test
     @mock.patch.dict(os.environ, {'FTS_COT_TO_DB': '1'})            # bool test
-    @mock.patch.dict(os.environ, {'FTS_MAINPATH': '/tmp/fts'})         # str test
+    @mock.patch.dict(os.environ, {'FTS_PERSISTENCE_PATH': '/tmp/fts'})         # str test
     def test_yaml_config_with_env_override(self):
         config = MainConfig.instance(config_file='/dev/null')
 
         assert(config.SSLCoTServicePort== 10000)
         assert(config.SaveCoTToDB == True)
-        assert(config.MainPath == '/tmp/fts')
+        assert(config.persistencePath == '/tmp/fts')
 
     def test_get_config_as_attribute(self):
         config = MainConfig.instance()
-        assert(config.MainPath == fr'{USERPATH}{PYTHON_VERSION}/dist-packages/FreeTAKServer')
+        assert(config.persistencePath == fr'/opt/fts/')
```

Comment:
> __[Pylint]__ Unnecessary parens after 'assert' keyword ([view](https://sider.review/gh/repos/238457011/pulls/420#issue-8341084))
>
> <details>
>
> | Rule | Severity | Message-id | Module | Obj |
> | ---- | -------- | ---------- | ------ | --- |
> | `superfluous-parens` | `convention` | C0325 | MainConfig_test | - |
>
> You can [close this issue](https://sider.review/gh/repos/238457011/pulls/420#issue-8341084) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

we have move all the tests out, for now so that should not be part of the commit